recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://github.com/dropbox/dropbox-sdk-python/archive/v{{ version }}.tar.gz
-  sha256: 010763f8536364af14b72d6c7380b839a5eadb3e7bfb99a2b76f6ab003f7989d
+  sha256: 54bfe5902b8cb26d2f70b2ee8bda92cd61a084d1672e19af3d60d04a36e8bee1
 
 build:
   number: 0
```

Comment:
Probably we should build it as **noarch** like upstream conda-forge https://github.com/conda-forge/dropbox-feedstock/blob/master/recipe/meta.yaml